index-graphclust.md

```diff
@@ -61,24 +76,26 @@ Below is a coarse-grained correspondence list of GraphClust2 tool names with eac
 |8,9| [Report results](https://graphclust.usegalaxy.eu/?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/graphclust_postprocessing/glob_report/0.5) and [conservation evaluations](https://graphclust.usegalaxy.eu/?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam%2Fgraphclust_aggregate_alignments/graphclust_aggregate_alignments/0.1) | Collect final clusters and create example alignments of top cluster members|
 {: .table.table-striped}
 
-### Input:
-The input to the workflow is a set of putative RNA sequences in FASTA format. Inside the `sample_data` directory you can find examples of the input format. In this case the data is from a benchmark set based on Rfam 12.0 and additionally is optionally labeled with reference Rfam family members.
+### Input
+The input to the workflow is a set of putative RNA sequences in FASTA format. Inside the `data` directory within the repository, you can find examples of the input format. 
```

Comment:
Why not a Galaxy history that you share?

Reply:
My personal account is getting so dirty with too many WFs and history items. As now there is a separate user account for graphclust, I will share new histories from that one. The WFs are already shared from the new account.